src/main/java/com/salesforce/dynamodbv2/mt/mappers/MtAmazonDynamoDb.java

```diff
@@ -152,6 +152,7 @@ public MtStreamDescription withRecordProcessorFactory(IRecordProcessorFactory re
     /*
      * MtAmazonDynamoDb-specific methods
      */
+    @Deprecated
     List<MtStreamDescription> listStreams(IRecordProcessorFactory factory);
```

Comment:
I plan to remove this method and its implementation & tests once we have switched ZOS over to use the standard listStreams method.